### PR TITLE
Load data

### PR DESCRIPTION
**Pull request by peterampazzo**

### CODE REVIEW

File: src/data/make_dataset.py [hunk 1]
```diff
@@ -3,32 +3,34 @@
 import logging
 from pathlib import Path
 from dotenv import find_dotenv, load_dotenv
+import torch
+import torchvision
```
Comment (alessandro-canevaro) [inline src/data/make_dataset.py:7]: Remember to add `torchvision` to requirements.txt

**Review by alessandro-canevaro — APPROVED** [review]

**Review by Loic-Ker — APPROVED** [review]

**Comment by Loic-Ker** [comment]
I add the cnn network and the training loop :). I tried to add datas inside data/raw (as one folder per letter) and then I ran make_dataset.py and train_model.py and it worked

File: src/models/conv_nn.py [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
-        self.cnn_layers = None
+        self.cnn_layers = nn.Sequential(
+        nn.Conv2d(in_channels=3, out_channels=32, kernel_size=8, stride=4), nn.ReLU(),
+        nn.Conv2d(in_channels=32, out_channels=64, kernel_size=4, stride=2), nn.ReLU(),
+        nn.Conv2d(in_channels=64, out_channels=64, kernel_size=3, stride=1), nn.ReLU())
 
-        self.linear_layers = None
+        self.flatten = nn.Flatten()
+        self.linear_layers = nn.Sequential(nn.Linear(in_features=36864, out_features=1024),nn.ReLU(),
+        nn.Linear(in_features=1024, out_features=512),nn.ReLU(),nn.Linear(in_features=512, out_features=3))
```
Comment (erlahrafnkels) [inline src/models/conv_nn.py]: Personally, I think I would indent lines 11-17 - but if you guys prefer it like this then it's okay too :)

**Review by erlahrafnkels — COMMENTED** [review]
LGTM, just added one formatting comment :)

**Review by alessandro-canevaro — APPROVED** [review]

File: src/models/conv_nn.py (hunk 2 above)
Comment (peterampazzo) [reply src/models/conv_nn.py]: fixed with `black` 😄

**Pull request merged**